docs/source/jupyter.rst

```diff
@@ -22,6 +14,25 @@ Dask-CHTC provides a way to run a Jupyter notebook server on a CHTC submit node.
     how to run here. We recommend skimming this page, then reading about
     port forwarding, then coming back here to try out the commands in full.
 
+.. warning::
+
+    **Do not** run Jupyter notebook servers on CHTC submit nodes except through
+    the process described on this page.
+
+You may want to interact with your Dask cluster through a
+`Jupyter notebook <https://jupyter.org/>`_.
+Dask-CHTC provides a way to run a Jupyter notebook server on a CHTC submit node.
+
+
+.. _Jupyter install documentation: https://jupyter.org/install
+
+.. warning::
+
+   Jupyter must be installed. This amounts to following the
+   `Jupyter install documentation`_, which amounts to running
+   ``conda install -c conda-forge jupyterlab`` or adding `jupyter` to the
+   ``environment.yml`` file.
```

Comment:
Nitpick: can we not use "amounts" twice here? I'd also say that we should prefer `conda install` without `-c conda-forge` unless the package is actually not available in the standard channel.

Reply:
https://jupyter.org/install recommend `conda install -c conda-forge jupyterlab`. However, it looks like it's available on the defaults channel (`conda install jupyter`). I'll make that edit.